medcat/relation_extraction.py

```diff
@@ -0,0 +1,631 @@
+import logging
+import os
+import numpy
+import spacy
+import logging
+import pandas
+import torch
+
+import torch.nn
+import pickle
+import dill
+import regex as re
+from torch.nn.utils.clip_grad import clip_grad_norm_
+import torch.optim
+import torch
+import torch.nn as nn
+from datetime import date, datetime
+from itertools import combinations
+from torch.nn.modules.module import T
+from torch.nn.utils.rnn import pad_sequence
+from transformers import BertConfig
+
+from transformers.configuration_utils import PretrainedConfig
+from itertools import permutations
+from pandas.core.series import Series
+
+from medcat.preprocessing.tokenizers import TokenizerWrapperBERT
+
+from spacy.tokens import Doc
+from typing import Dict, Iterable, List, Set, Tuple
+from transformers import AutoTokenizer
+
+from tqdm.autonotebook import tqdm
+
+from medcat.utils.models import BertModel_RelationExtracation
```

Comment:
This import is broken,
should be from medcat.utils.relation_extraction import BERTRElationExtraction